hive/server/condenser_api/call.py

```diff
@@ -93,4 +102,12 @@ def _strict_query(params, ignore_key=None):
     elif method == 'get_replies_by_last_update':
         return await get_replies_by_last_update(*_strict_list(params, 3))
 
-    raise Exception("unknown method: {}.{}({})".format(api, method, params))
```

Comment:
This gets pulled for the conditional with return?  is there any other lifeboats we should think about?  and not sure on py styling but sometimes if i get 1-2 conditions deep i'll go case/switch.

Reply:
It's not getting pulled, there's just a few more routes added and if none match it will throw `ApiError` at the end of the method. And I would definitely use switch if python supported such a feature!